pathd/path_cli.c

```diff
@@ -1092,7 +1092,25 @@ DEFPY_NOSH(show_debugging_pathd, show_debugging_pathd_cmd,
 	   "State of each debugging option\n"
 	   "pathd module debugging\n")
 {
-	/* nothing to do here */
+	vty_out(vty, "Path debugging status:\n");
+	path_ted_show_debugging(vty);
+	path_policy_show_debugging(vty);
+	return CMD_SUCCESS;
+}
+
+DEFPY (debug_path_policy,
+       debug_path_policy_cmd,
+       "[no] debug pathd policy",
+       NO_STR
+       DEBUG_STR
+       "path debugging\n"
+       "policy debugging\n")
+{
+	uint32_t mode = DEBUG_NODE2MODE(vty->node);
+	bool no_debug = (no != NULL);
```

Comment:
Could you explain why this is needed? `bool no_debug = no` simply?

Reply:
right. I just fixed it.